src/lib/tooltipDescriptions.ts

```diff
@@ -4,7 +4,7 @@ export enum TOOLTIP_DESCRIPTIONS {
   CURRENT_ORACLE_PRICE = "The price that will be monitored to trigger the order posting.",
   CURRENT_MARKET_PRICE = "The current price of the token pair in the market.",
   ORDER_CREATION = "The transaction that created this stop loss order.",
-  ORDER_HASH = "The onchain transaction for creating this order.",
+  ORDER_HASH = "The hash of the stop loss order on the Composable CoW framework. Each order for each user has a unique hash.",
```

Comment:
isn't this what they call order ID?

Reply:
No, the order ID is when it is posted on the Orderbook API. The hash is a parameter to find the order on the Composable CoW.